README.md

```diff
@@ -153,20 +164,14 @@ For all:
 
 - the index.html page (currently 1 occurrence in page text)
 - the appropriate menu entry(s) in default.html (only number releases i.e. 5 -> 6)
-- the “title:" in relevant HTML pages - getting-started-5.html and 
-  importing-5.html (only point releases)
-- the heading in the relevant Word .doc file (only point releases)
 - the version number in two screenshots in the relevant .graffle file -
   Downloads and Zips. Note: the version numbers are added as editable text in
   the Version 5.x Omnigraffle canvases. The new versions of the screenshots
   are saved with the same name over-writing the previous version.
 - the new versions of the screenshots need to be placed in the images/ folder.
-- new versions of the PDF need to be generated
-- a new ZIP archive file for all the PDFs for that version generated i.e one
-for  5.x (with the appropriate sections for the  new versions
-included).
-- the previous version ZIP archive file becomes the reference archive for that
-version - linked to from the ”Previous” page.
+- a new ZIP archive file for all the PDFs for that version generated i.e one for  5.x (with the appropriate sections for the  new versions included).
```

Comment:
I can't quite make sense of this sentence; the confusion's around "generated". (If it's just me, though, don't worry.)